code/modules/mapping/station_datums.dm

```diff
@@ -27,7 +27,7 @@
 	fluff_name = "NSS Seraphim"
 	technical_name = "EmeraldStation"
 	map_path = "_maps/map_files/stations/emeraldstation.dmm"
-	webmap_url = "https://www.youtube.com/watch?v=dQw4w9WgXcQ&pp=ygUIcmlja3JvbGw%3D"
+	webmap_url = "https://affectedarc07.co.uk/emerald.html"
```

Comment:
For everyone else here
---
This will change when a real webmap is done, said webmap cant be done until the map is in the repo fully.

Reply:
I can't believe I got rickrolled by a code change.

Reply:
> I can't believe I got rickrolled by a code change.

Do you not instinctly recognise `qQw4`?